build/dsc.rake

```diff
@@ -60,7 +60,36 @@ eod
         cmd += "git checkout #{ENV['DSC_REF']} &&" if ENV['DSC_REF']
         cmd += "git submodule update --init --recursive"
 
-        sh cmd
+        sh cmd unless File.exists?(dsc_resources_path_tmp)
+
+        puts "Creating sha file"
+        File.open("#{default_dsc_module_path}/dsc_resource_release_tags.md", 'w+') do |file|
+          file.write("## SHA tags for each DSC Resource repo\n")
+          file.write("\n")
+          file.write("DSC Resource | SHA | Release Tag\n")
+          file.write("------------ | --- | -----\n")
+          
+          Dir["#{dsc_resources_path_tmp}/xDSCResources/*"].each do |path|
+            if File.directory?(path)
+              puts "Finding latest tag for #{path}"
+              FileUtils.cd(path) do
+                begin
+                  repo = Git.open(path, :log => Logger.new(STDOUT))
```

Comment:
I did originally, but met resistance when i couldn't figure out how to return the output from the git command using `sh`